### DISCUSSION TITLE
Not receiving mail in production ?

### DISCUSSION THREAD

**Discussion by hakerinjo** [pocketbase/pocketbase#2034] [Q&A]
Hello, 
I got identical setup for email in http://localhost:8090/_/#/settings/mail 
And when I click 'Send test mail' from local it works, but when I click it in production, it also says email successfully sent, but I never get that mail from production env, not even in spam.. 

My email server host is smtp.office365.com 

Is there an way I can debug this ?

Any idea what's happening ?

**Comment by ganigeorgiev** [comment]
Once the message is sent to the mail server it is up to the mail server to deliver it to the recipient. We don't have control over this. It is possible that the message is being blocked for triggering some spam filter (eg. failing a dkim check or similar).

To debug it, you can try looking through the logs of your mail server.

In the 365 docs there is something called ["Message trace"](https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/message-trace-scc?view=o365-worldwide#open-message-trace:). I'm not sure if you have access to it and whether it is applicable for your use case, but you can give it a try:

> In the Microsoft 365 Defender portal at https://security.microsoft.com/, go to **Email & collaboration > Exchange message trace**. To go directly to the message trace page, use https://admin.exchange.microsoft.com/#/messagetrace.
